### PR TITLE
Enable creation and consumption of a 'coherent' pointer

### PR DESCRIPTION
**Pull request by ghost**
Enable creation and consumption of a 'coherent' pointer
Write a 'latest.coherent.version' file at blob storage so users have a 'coherent' option to SDK installations.
Modify install script adding a 'coherent' version option.

Other:
Remove the unused 'debug_symbols' option.
Expand explanation of 'Channel' and 'Version' options.

@dotnet/dotnet-cli

@MattGertz

### CODE REVIEW

**Comment by johnbeisner** [comment]
@dotnet/dotnet-cli 
Can someone take a look at this PR and comment.

**Comment by livarcocc** [comment]
@mikeharder @Eilon

**Review by wli3 — APPROVED** [review]

**Comment by muratg** [comment]
This looks good to me.

File: scripts/obtain/dotnet-install.ps1 [hunk 1]
```diff
@@ -464,14 +478,17 @@ if ($free.Freespace / 1MB -le 100 ) {
 }
 
 $ZipPath = [System.IO.Path]::GetTempFileName()
-Say "Downloading $DownloadLink"
+Say-Verbose "Zip path: $ZipPath"
```
Comment (livarcocc) [inline scripts/obtain/dotnet-install.ps1]: Why did we add this Zip-path message and the one below?

File: scripts/obtain/dotnet-install.ps1 [hunk 2]
```diff
@@ -207,15 +212,20 @@ function GetHTTPResponse([Uri] $Uri)
 }
 
 
-function Get-Latest-Version-Info([string]$AzureFeed, [string]$Channel) {
+function Get-Latest-Version-Info([string]$AzureFeed, [string]$Channel, [bool]$Coherent) {
```
Comment (livarcocc) [inline scripts/obtain/dotnet-install.ps1]: $Coherent does not need to be in the params above?

File: scripts/obtain/dotnet-install.ps1 (hunk 1 above)
Comment (livarcocc) [inline scripts/obtain/dotnet-install.ps1]: Nvm, I see now, you made that part of version, that makes sense.

File: scripts/obtain/dotnet-install.ps1 (hunk 2 above)
Comment (livarcocc) [inline scripts/obtain/dotnet-install.ps1]: Nvm, I see now, you made that part of version, that makes sense.

File: scripts/obtain/dotnet-install.ps1 (hunk 1 above)
Comment (johnbeisner) [inline scripts/obtain/dotnet-install.ps1]: To align behaviors with the shell script.

**Pull request merged**